api/v1/swagger.yaml

```diff
@@ -3094,6 +3094,17 @@ paths:
             Filter by parentAbsolutePath; asterisk (*) wildcards ok.
           required: false
           type: string
+        -
+          name: tags
+          in: query
+          items:
+            type: string
+            maxLength: 60
```

Comment:
ok we can get this through as is, but i think we want to follow up with story to allow `v1/subjects?tags=tag1,tag2,tag3` or `v1/subjects?tags=-tag10,tag11` for parity with the analogous functionality on the /hierarchy endpoint